.github/workflows/release-nightly.yml

```diff
@@ -0,0 +1,48 @@
+name: Release Nightly
```

Comment:
If this is copied from Rsbuild repository, we need to ensure nightly workflow is still being used. As for https://github.com/web-infra-dev/rsbuild/actions/workflows/release-nightly.yml, the last it run was 9 months ago.

Reply:
nightly seems no need now, rsbuild doesnt publish nightly for a long time, maybe we can delete it.

Reply:
Yeah, canary / alpha(beta) and latest is enough for us.